torch/testing/_internal/common_nn.py

```diff
@@ -4660,6 +4671,7 @@ def __init__(self, *args, **kwargs):
         self.check_inplace = kwargs.get('check_inplace', False)
         self.check_gradgrad = kwargs.get('check_gradgrad', True)
         self.skip_double = kwargs.get('skip_double', False)
+        self.test_cpu = kwargs.get('test_cpu', True)
```

Comment:
I didn't see any way to disable CPU for the non-deterministic alert tests for operations in the nn module, so I had to add this.